src/Bicep.Core/TypeSystem/TypeSymbolValidationFlags.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+
+namespace Bicep.Core.TypeSystem
+{
+    [Flags]
+    public enum TypeSymbolValidationFlags
```

Comment:
>TypeSymbolValidationFlags [](start = 16, length = 25)

Can you comment on the enum members to describe what their effect is?

Reply:
Also wonder if a more self describing name would be TypeErrorLevel with Error and Warning values.

---
In reply to: [497790260](https://github.com/Azure/bicep/pull/578#discussion_r497790260) [](ancestors = 497790260)

Reply:
I'm planning to use this as a more general-purpose type attribute, so have left as-is. I've renamed `Permissive` to `WarnOnTypeMismatch` to be more explicit and added comments.